### PR TITLE
blog/2023-04-13-rse-on-mastodon: state of RSE on mastodon

### PR DESCRIPTION
**Pull request by rkdarst**
- Quick post based on thread I started, please revise/improve before
  publication.
- Review: needs a detailed read + some editorial checking for both
  accuracy and message.

### CODE REVIEW

File: content/blog/2023-04-13-rse-on-mastodon.md
Comment (bielsnohr) [inline content/blog/2023-04-13-rse-on-mastodon.md:35]: Perhaps worth a mention that `#RSEng` was the primary hastag on Twitter for RSE-related stuff

File: content/blog/2023-04-13-rse-on-mastodon.md
Comment (bielsnohr) [inline content/blog/2023-04-13-rse-on-mastodon.md:30]: I think getting a category at <https://fedi.directory/> as you mention below is the best option for this. It will be inherently more visible than any bespoke list we might start, and it avoids any decisions about where such a list would be hosted and maintained.  (This is @mattasdata from Mastodon btw. Need to reconcile my different usernames :stuck_out_tongue: )

File: content/blog/2023-04-13-rse-on-mastodon.md
Comment (bielsnohr) [inline content/blog/2023-04-13-rse-on-mastodon.md:63]:
```suggestion
One could ask if it's better to have a common server, or be more
```

File: content/blog/2023-04-13-rse-on-mastodon.md
Comment (bielsnohr) [inline content/blog/2023-04-13-rse-on-mastodon.md:71]: In my opinion, there is a better chance of an RSE-focused server being set up because there are actual RSE organisations. "Scientific computing" is a much more nebulous and dispersed domain, and I suspect the only people who would actually sign up to it would be RSEs anyway (hence why not just make an RSE instance).

Some next actions would be making a formal proposal to some or all of the RSE organisations. There is the international RSE council which would be a catch-all: https://researchsoftware.org/council.html

File: content/blog/2023-04-13-rse-on-mastodon.md
Comment (alansill) [inline content/blog/2023-04-13-rse-on-mastodon.md:67]: The primary advantage of having a server focused on any topic, for example RSE, is the ability to browse the local timeline without needing to wait for federation of related servers to make relevant posts local. Of course, any server can be used as a home base in the Fediverse, and any account can follow posts using a given hasttag by simply following that hashtag, so having a server for a community is largely a convenience, and doing so provides a natural gathering mechanism for people interested in a given topic.

File: content/blog/2023-04-13-rse-on-mastodon.md
Comment (alansill) [inline content/blog/2023-04-13-rse-on-mastodon.md:60]: There is overlap between the HPC and RSE communities, and the initial community survey ([ref](https://survey.hpc.social/results/hpc-social-design-survey.html)) for creation of the HPC.social Mastodon server showed a significant interest in exchanging information on software.

File: content/blog/2023-04-13-rse-on-mastodon.md
Comment (hturner) [reply content/blog/2023-04-13-rse-on-mastodon.md:71]: Also "scientific computing" would not be a good fit for RSEs who work in social science, humanities, multidisciplinary projects, etc.

When thinking of running an instance you need to think about governance, moderation, sustainability, etc - attaching to existing an community or (group of) organisation(s) should help with this.

File: content/blog/2023-04-13-rse-on-mastodon.md
Comment (hturner) [inline content/blog/2023-04-13-rse-on-mastodon.md:78]: There was some discussion on Slack at the end of 2022. More people have moved to Mastodon since then - it would be good to have more institutions/organizations engaging on Mastodon, but I'm not sure if anyone's ready to run an instance.

File: content/blog/2023-04-13-rse-on-mastodon.md
Comment (danielskatz) [reply content/blog/2023-04-13-rse-on-mastodon.md:35]: Could also say that when both are used, #RSE is the people and #RSEng is the activity, though #RSE is often used for both.

File: content/blog/2023-04-13-rse-on-mastodon.md
Comment (danielskatz) [reply content/blog/2023-04-13-rse-on-mastodon.md:44]: I don't think it's so useful to have a term for scientific computing that is highlighted. It's the research computing that matters here, whether it's science, engineering, humanities, etc.

File: content/blog/2023-04-13-rse-on-mastodon.md
Comment (rantahar) [reply content/blog/2023-04-13-rse-on-mastodon.md:35]: Cannot really comment on this, but it's probably better to publish this rather than wait. Keep as it is?

File: content/blog/2023-04-13-rse-on-mastodon.md
Comment (rantahar) [inline content/blog/2023-04-13-rse-on-mastodon.md:25]:
```suggestion
* Research Software Alliance: <https://fosstodon.org/@researchsoft>
* Netherlands eScience Center: <https://fosstodon.org/@eScienceCenter@akademienl.social>
* ... ?
```

File: content/blog/2023-04-13-rse-on-mastodon.md
Comment (rantahar) [reply content/blog/2023-04-13-rse-on-mastodon.md:30]: Does this mean removing this section for now, or keeping the TODO?

File: content/blog/2023-04-13-rse-on-mastodon.md
Comment (rantahar) [reply content/blog/2023-04-13-rse-on-mastodon.md:78]: There are some interesting points in this PR, but I cannot convert them to changes in the blog post. This section would probably be the best place for them in this blog post. Unfortunately they will be effectively lost after merging the PR.

**Review by rantahar — APPROVED** [review]

File: content/blog/2023-04-13-rse-on-mastodon.md
Comment (rkdarst) [inline content/blog/2023-04-13-rse-on-mastodon.md:30]:
```suggestion
TODO: We should try to get a category in <https://fedi.directory/> rather than make our own list.
```

File: content/blog/2023-04-13-rse-on-mastodon.md
Comment (rkdarst) [inline content/blog/2023-04-13-rse-on-mastodon.md:37]:
```suggestion
  as Twitter).  There doesn't seem to be much activity there.  #RSEng was
  the primary tag from from Twitter.  Sometimes #RSE is the people and
  #RSEng is the activity.
```

File: content/blog/2023-04-13-rse-on-mastodon.md
Comment (rkdarst) [inline content/blog/2023-04-13-rse-on-mastodon.md:60]:
```suggestion
([ref](https://mast.hpc.social/@AlanSill/110147881007346377)).  There is overlap
between the HPC and RSE communities, and the initial community survey (ref)
for creation of the HPC.social Mastodon server showed a significant interest in
exchanging information on software topics.

Many
```

File: content/blog/2023-04-13-rse-on-mastodon.md
Comment (rkdarst) [inline content/blog/2023-04-13-rse-on-mastodon.md:66]:
```suggestion
fedi-science](https://fediscience.org/server-list.html). The primary advantage
of a topic-driven server is a better local timeline, which will be populated with
the posts of everyone on that server.  While following hashtags/etc. is similar,
the local server makes this more natural.
```

File: content/blog/2023-04-13-rse-on-mastodon.md
Comment (rkdarst) [inline content/blog/2023-04-13-rse-on-mastodon.md:71]:
```suggestion
software, etc. people.  Yet, this is also too broad and nebulous and without
organizational backing so comments were negative on this idea.)
```

File: content/blog/2023-04-13-rse-on-mastodon.md
Comment (rkdarst) [reply content/blog/2023-04-13-rse-on-mastodon.md:71]: I agree with the comments that this is too broad and we couldn't get someone to run it.  Commented to say this.

File: content/blog/2023-04-13-rse-on-mastodon.md
Comment (rkdarst) [reply content/blog/2023-04-13-rse-on-mastodon.md:78]: I added them as changes.  I think the comments would be preserved, I will

**Review by rkdarst — COMMENTED** [review]
Well, this took an excessively long time to get back to.  Let's hope this isn't obsolete by now.  I added suggestions adding most of the comments into the text itself.

Also it's summertime so some relevant people may be on vacation.  When is a good time to merge?

File: content/blog/2023-04-13-rse-on-mastodon.md
Comment (danielskatz) [reply content/blog/2023-04-13-rse-on-mastodon.md:71]: The HPC community has https://hpc.social which includes https://mast.hpc.social/about, supported by organizational contributions,

File: content/blog/2023-04-13-rse-on-mastodon.md
Comment (danielskatz) [inline content/blog/2023-04-13-rse-on-mastodon.md:22]:
```suggestion
* DE RSE: <https://mastodon.social/@de_rse>
```

File: content/blog/2023-04-13-rse-on-mastodon.md
Comment (danielskatz) [reply content/blog/2023-04-13-rse-on-mastodon.md:25]: The Netherlands one should be
`https://akademienl.social/@eScienceCenter`

File: content/blog/2023-04-13-rse-on-mastodon.md
Comment (hturner) [inline content/blog/2023-04-13-rse-on-mastodon.md:35]:
```suggestion
* **#RSEng** and **#RSE** have been seen, but #RSEng is preferred
```

File: content/blog/2023-04-13-rse-on-mastodon.md
Comment (hturner) [reply content/blog/2023-04-13-rse-on-mastodon.md:37]: “(same as Twitter)” not needed given next sentence explains further.

**Review by hturner — COMMENTED** [review]
I suggested some minor edits. I think it is good to go once everyone’s suggested changes are made.

File: content/blog/2023-04-13-rse-on-mastodon.md
Comment (rkdarst) [inline content/blog/2023-04-13-rse-on-mastodon.md:38]:
```suggestion
  ([ref](https://fosstodon.org/@HeathrTurnr/110147262319789502).
```

File: content/blog/2023-04-13-rse-on-mastodon.md
Comment (rkdarst) [inline content/blog/2023-04-13-rse-on-mastodon.md:26]:
```suggestion
* Netherlands eScience Center: <https://akademienl.social/@eScienceCenter>
* RSE Midlands <https://mastodon.social/@RSE_Midlands>
```

**Comment by rkdarst** [comment]
if anyone wants tob e listed as an author, let me know.  I will update it to say "[with many ideas and contributions from commenters in the Github pull request](https://github.com/nordic-rse/nordic-rse.github.io/pull/412)" to credit everyone who took part.

File: content/blog/2023-04-13-rse-on-mastodon.md
Comment (rkdarst) [reply content/blog/2023-04-13-rse-on-mastodon.md:80]: (added to text)

**Pull request merged**
